tests/cpp/testobject/PdxClassV1.cpp

```diff
@@ -560,69 +550,69 @@ std::string PdxTypesIgnoreUnreadFieldsV1::toString() const {
   sprintf(idbuf, "PdxTypesV1R1:[ m_i1=%d ] [ m_i2=%d ] [ m_i3=%d ] [ m_i4=%d ]",
           m_i1, m_i2, m_i3, m_i4);
   return idbuf;
- }
-
- /************************************************************
-  *  PdxVersionedV1
-  * *********************************************************/
-
- PdxVersionedV1::PdxVersionedV1() {}
-
- void PdxVersionedV1::init(int32_t size) {
-   m_char = 'C';
-   m_bool = true;
-   m_byte = 0x74;
-   m_int16 = 0xab;
-   m_int32 = 0x2345abdc;
-   m_long = 324897980;
-   m_float = 23324.324f;
-   m_double = 3243298498;
-
-   m_string = (char *)"gfestring";
-
-   m_boolArray = new bool[3];
-   m_boolArray[0] = true;
-   m_boolArray[1] = false;
-   m_boolArray[2] = true;
-
-   m_charArray = new char[2];
-   m_charArray[0] = 'c';
-   m_charArray[1] = 'v';
-
-   m_dateTime = nullptr;
-
-   m_int16Array = new int16_t[size];
-   m_int32Array = new int32_t[size];
-   m_longArray = new int64_t[size];
-   m_floatArray = new float[size];
-   m_doubleArray = new double[size];
-
-   for (int i = 0; i < size; i++) {
-     m_int16Array[i] = 0x2332;
-     m_int32Array[i] = 34343 + i;
-     m_longArray[i] = 324324L + i;
-     m_floatArray[i] = 232.565f + i;
-     m_doubleArray[i] = 23423432 + i;
-     // m_stringArray[i] = String.Format("one{0}", i);
-   }
-
-   boolArrayLen = 0;
-   byteArrayLen = 0;
-   shortArrayLen = 0;
-   intArrayLen = 0;
-   longArrayLen = 0;
-   doubleArrayLen = 0;
-   floatArrayLen = 0;
-   strLenArray = 0;
- }
-
- PdxVersionedV1::PdxVersionedV1(int32_t size) {
-   init(size);
-   LOGDEBUG("PdxVersioned 1");
- }
-
- PdxVersionedV1::~PdxVersionedV1() {
-   // TODO Auto-generated destructor stub
+}
+
+/************************************************************
+ *  PdxVersionedV1
+ * *********************************************************/
+
+PdxVersionedV1::PdxVersionedV1() {}
+
+void PdxVersionedV1::init(int32_t size) {
+  m_char = 'C';
+  m_bool = true;
+  m_byte = 0x74;
+  m_int16 = 0xab;
+  m_int32 = 0x2345abdc;
+  m_long = 324897980;
+  m_float = 23324.324f;
+  m_double = 3243298498;
+
+  m_string = "gfestring";
+
+  m_boolArray = new bool[3];
+  m_boolArray[0] = true;
+  m_boolArray[1] = false;
+  m_boolArray[2] = true;
+
+  m_charArray = new char[2];
+  m_charArray[0] = 'c';
+  m_charArray[1] = 'v';
+
+  m_dateTime = nullptr;
+
+  m_int16Array = new int16_t[size];
+  m_int32Array = new int32_t[size];
+  m_longArray = new int64_t[size];
+  m_floatArray = new float[size];
+  m_doubleArray = new double[size];
+
+  for (int i = 0; i < size; i++) {
+    m_int16Array[i] = 0x2332;
+    m_int32Array[i] = 34343 + i;
+    m_longArray[i] = 324324L + i;
+    m_floatArray[i] = 232.565f + i;
+    m_doubleArray[i] = 23423432 + i;
+    // m_stringArray[i] = String.Format("one{0}", i);
```

Comment:
This was commented out in the original code, but thought I'd point it out.  Not used?  Not supported?